ethcore/src/ethereum/ethash.rs

```diff
@@ -73,6 +75,7 @@ impl From<ethjson::spec::EthashParams> for EthashParams {
 			difficulty_hardfork_transition: p.difficulty_hardfork_transition.map_or(0x7fffffffffffffff, Into::into),
 			difficulty_hardfork_bound_divisor: p.difficulty_hardfork_bound_divisor.map_or(p.difficulty_bound_divisor.into(), Into::into),
 			bomb_defuse_transition: p.bomb_defuse_transition.map_or(0x7fffffffffffffff, Into::into),
+			bad_gas_compatibility_mode_limit: p.bad_gas_compatibility_mode_limit.map_or(0, Into::into),
```

Comment:
This will break all private chains after upgrade. Maybe leave it disabled by default and just update sample and documentation specs to enable it?